crates/sargon/src/core/error/common_error.rs

```diff
@@ -734,6 +734,9 @@ pub enum CommonError {
 
     #[error("Unable to make {amount} transfers in one single transaction")]
     MaxTransfersPerTransactionReached { amount: u64 } = 10208,
+
+    #[error("Transaction Manifest class is forbidden: {}", class.name())]
+    ForbiddenManifestClass { class: DetailedManifestClass } = 10209,
```

Comment:
Some derived manifest classes can only happen as result of the interaction coming from the Wallet.